builders/pallets-precompiles/pallets/utility.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Utility Pallet
+description: Learn about the available extrinsics for the utility pallet on Moonbeam and how to interact with them using Polkadot.js Apps and the Polkadot.js API.
+keywords: utility, batch, substrate, pallet, moonbeam, polkadot
+---
+
+# The Utility Pallet
```

Comment:
I wonder if we should add examples or something as this feels a bit empty. We could:

1. Explain a bit more derivative accounts (check the xcm-transactor page-> http://127.0.0.1:8000/builders/xcm/xcm-transactor/#general-xcm-definitions) We could then add a reference from the xcm-transactor page to here
2. Do an example with the batching?